Jenkinsfile

```diff
@@ -62,6 +62,7 @@ def gitCommitId
 def gitRepoUrl
 def awsRegion = 'us-east-1'
 def verboseMaven = params.verbose_mvn_logging
+def migratorRunbookUrl = "https://github.com/CMSgov/beneficiary-fhir-data/blob/master/runbooks/how-to-recover-from-migrator-failures.md"
```

Comment:
This is a low risk change, but I'd **still** like to fully test this prior to merging.

Reply:
OK!

1. Out of band, I created the migrator instance in `test`, and left a populated SQS queue in tact (attempting to trigger the Undeployable State error).
2. I attempted to deploy to `test` via Jenkins on PR-1475 and I was met with the expected error and link to a runbook (although it's hardcoded to master, which doesn't yet exist...)
3. Using the runbook corrective steps, I've re-deployed to `test` via Jenkins, and the deployment is functioning as expected.